packages/artifact/src/internal/shared/artifact-twirp-client.ts

```diff
@@ -56,16 +56,14 @@ class ArtifactHttpClient implements Rpc {
     const headers = {
       'Content-Type': contentType
     }
-    info(`Making request to ${url} with data: ${JSON.stringify(data)}`)
-
     try {
       const response = await this.retryableRequest(async () =>
         this.httpClient.post(url, JSON.stringify(data), headers)
       )
       const body = await response.readBody()
       return JSON.parse(body)
     } catch (error) {
-      throw new Error(error.message)
+      throw new Error(`Failed to ${method}: ${error.message}`)
```

Comment:
This will end up looking like `Failed to CreateArtifact: <error message>` or `Failed to FinalizeArtifact: <error message>`